rudder-webapp/SOURCES/Makefile

```diff
@@ -57,11 +57,15 @@ localdepends: ./rudder-sources ./maven/bin/mvn ./rudder-users.xml ./rudder-doc .
 ./rudder-users.xml: ./rudder-sources
 	cp ./rudder-sources/rudder/rudder-web/src/main/resources/demo-rudder-users.xml ./rudder-users.xml
 
+./.gitignore: ./configuration-repository-gitignore
+	cp ./configuration-repository-gitignore ./.gitignore
```

Comment:
That is hard to acheive in debian packaging. There is a special dh_* helper that does the rename for you for init scripts (and similar things like crons), but dh_install doesn't support renaming by itself :(